jxmapviewer2/src/main/java/org/jxmapviewer/viewer/GeoBounds.java

```diff
@@ -33,10 +33,10 @@ public GeoBounds(double minLat, double minLng, double maxLat, double maxLng)
      */
     public GeoBounds(Set<GeoPosition> geoPositions)
     {
-        if (geoPositions == null || geoPositions.size() < 2)
+        if (geoPositions == null || geoPositions.size() == 0)
```

Comment:
It still needs to receive two points, no?

Reply:
If I have 2 or more points on a horizontal line/a latitude and I draw a bounding box around them, then I get a rectangle with a height of zero. If I have 2 or more points on a vertical line/a longitude and I draw a bounding box around them, then I get a rectangle with a width of zero. So if a height of zero and a width of zero is allowed, then zero values in both dimensions simultaneously could be allowed too, meaning it is possible to draw a bounding box with zero height and zero width around a single point. So only one GeoPosition is needed to draw a bounding box. If zoomToBestFit() is called with a zero sized bounding box MapViewer zooms to the most detailed zoom level. It works perfectly for me. Sorry for my late answer. My application which uses JXMapViewer is still heavily under construction, but the integration of JxMapViewer works perfectly with this simple modification.

Reply:
Ah, good point, thanks. I'll review it again